docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -0,0 +1,254 @@
+The CPU resource is defined in CPU units where one CPU unit is the equivalent of one CPU core. Fractional units can also be requested, in either decimals like `0.1` but also in terms of millicores where `100m` would be the same amount. The memory resource is defined in whole integers with the quantity suffixes
+`Ei, Pi, Ti, Gi, Mi, Ki`.
+
+If you do not specify any resources for a container the default resource request and limit will be applied as shown below. These resources are low on purpose, both to minimize the effects of over provisioning but also to make it obvious for end users that resources have not been specified. The keen-eyed may notice that
+a CPU limit is not set by default, this is on purpose and the reasons for it will be discussed later.
+
+```yaml
+resources:
+  request:
+    cpu: 50m
+    memory: 32Mi
+  limits:
+    memory: 256Mi
+```
+
+The scheduler will look at the cumulative resource requests across all containers in a Pod during the scheduling phase. The scheduler will exclude any Node which does not have capacity for the Pods resource request. Capacity is determined based on the total resources available in a Node minus the sum of all the
+resource requests of all Pods currently scheduled to the Node. This makes it possible to over provision as the resource request can be much larger than the actual consumption of the Pod. Efficient resource allocation is a constant battle between requesting enough resources to avoid under allocation while not
+requesting too much which would result in over allocation. A Pod may at times request more resource than any Node has capacity for, there are two possible outcomes for this situation. If the Pods resource request is less than a Nodes total available resources, a new Node will be added to the cluster. The Pod will
+however be considered unschedulable if the resource request exceeds the total resources available on a single node. In these cases either the resource request has to change or a new Node type has to be added to the cluster to cater to these needs.
+
+<img alt="Pod Scheduling" src={useBaseUrl("img/assets/xks/developer-guide/pod-scheduling.jpg")} />
+
+The resource limit defined for a Pod has no affect on the scheduling of a Pod. Limits instead comes into play for a Pod during runtime. Exceeding the resource limit for CPU and memory will have
+different affects. A Pod which exceeds the memory limit will be terminated with an out of memory error (OOM Error).  The Pod will after termination be started again, it may start to exceed the limit again which will result in another OOM error. These types of errors can either be resolved by having the application
+consume less memory alternatively increasing the memory limit. Without a memory limit a Pod would be able to continue consuming memory until the Node runs out. This would not only affect critical
+system processes that runs in the node but other Pods which may not even be able to consume the resources it requested.

```

Comment:
I think a unlimited deployment can basically sink the whole cluster assuming fortuitous scheduling and more pods than there are nodes? If so, this formulation could be sharpened further to underline that sane people fastens the seat belt before starting cars and sets limits before deploying pods.